pkg/relic/Makefile.dep

```diff
@@ -0,0 +1,2 @@
+USEMODULE += random
+USEMODULE += prng_xorshift
```

Comment:
For the record, this is supposed to use the slimmest PRNG in RIOT to seed the relic crypto PRNG. In future we would use an entropy interface here.